src/components/ebay-textbox/test/test.server.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-expressions */
+
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/server');
+const rootSelector = `span.textbox`;
+const inputSelector = `input.textbox__control`;
+const textareaSelector = `textarea.textbox__control`;
+
+test('renders default input textbox', context => {
+    const $ = testUtils.getCheerio(context.render());
+    expect($(rootSelector).length).to.eql(1);
+    expect($(inputSelector).length).to.eql(1);
+});
+
+test('renders input textbox with some value', context => {
+    const input = { value: 'this is texbox' };
+    const $ = testUtils.getCheerio(context.render(input));
+    expect($(inputSelector).val()).to.equal('this is texbox');
+});
+
+test('renders input textbox with a placeholder', context => {
+    const input = { placeholder: 'enter your input' };
+    const $ = testUtils.getCheerio(context.render(input));
+    expect($(inputSelector).attr('placeholder')).to.equal('enter your input');
+});
+
+test('renders a disabled input textbox', context => {
+    const input = { disabled: true };
+    const $ = testUtils.getCheerio(context.render(input));
+    expect(Boolean($(inputSelector).is(':disabled'))).to.be.true;
+});
+
+test('renders a input textbox with invalid/error state', context => {
+    const input = { invalid: true };
+    const $ = testUtils.getCheerio(context.render(input));
+    expect(Boolean($(inputSelector).prop('aria-invalid'))).to.be.true;
+});
+
+test('renders input textbox with a custom class', context => {
```

Comment:
Remove this test, and instead use the `testCustomClass` util. Put it alongside the `testHtmlAttributes` test at the bottom. You'll see its usage in other component tests.